### ISSUE TITLE
Add a bulk download option for all files

### ISSUE THREAD

**Issue by KingSupernova31** [lunakv/academyruins-api#56]
The API currently allows you to download files by set code or date, but if I want all of them there's no easy way to do that.

**Comment by lunakv** [comment]
You can view set information about all available sets using the `/metadata/{cr|mtr|ipg}` endpoint, then iterate over the info from the response to get all hosted files. 

Note that these endpoints are unpublished and subject to future change.

**Comment by KingSupernova31** [comment]
Fair enough. I'll leave this open in case you want to eventually add that endpoint to the docs.
